gatsby/static/web.config

```diff
@@ -30,6 +31,8 @@
 					<add key="/treatment-summary/bacillus-calmette-guérin-vaccine-2.html" value="/treatment-summaries/bacillus-calmette-guerin-vaccine/" />
 					<add key="/medicinal-forms/bacillus-calmette-guérin.html" value="/drugs/bacillus-calmette-guerin/medicinal-forms/" />
 					<add key="/medicinal-forms/bacillus-calmette-guérin-vaccine.html" value="/drugs/bacillus-calmette-guerin-vaccine/medicinal-forms/" />
+					<!-- Internal links embedded in feed content -->
+					<add key="/guidance/PHP97237" value="/medicines-guidance/adverse-reactions-to-drugs/" />
```

Comment:
Please can you remove these 2 lines, now that we've added the `@html` directive to the GQL schema?

Reply:
Aha, thanks!